skate-plugin/src/main/kotlin/com/slack/sgp/intellij/SqlDelightErrorHandler.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2016 Slack Technologies, LLC
```

Comment:
We'll need to preserve the original copyright header

Reply:
36c089b8d35bdf57ba506ec09db5c411b331fa27